solana/solana-ibc/programs/solana-ibc/src/lib.rs

```diff
@@ -767,7 +863,7 @@ pub struct SendTransfer<'info> {
     #[account(mut)]
     token_mint: Option<Box<Account<'info, Mint>>>,
     #[account(init_if_needed, payer = sender, seeds = [
-        ESCROW, port_id.as_bytes(), channel_id.as_bytes(), hashed_base_denom.as_ref()
+        ESCROW, port_id.as_bytes(), channel_id.as_bytes(), hashed_full_denom.as_ref()
```

Comment:
If we’re hashing full denome do we still need port and channel in the seeds?

Reply:
Yes im pushing a fix for this